docs/design-docs/k8s-engines-on-prem/k8s-engines-on-prem.md

```diff
@@ -0,0 +1,361 @@
+kube-system   pod/coredns-58b9c97c99-7qgnm                     1/1     Running   0          55s
+kube-system   pod/coredns-58b9c97c99-9fx4m                     1/1     Running   0          51s
+kube-system   pod/dns-autoscaler-77c78db666-w2kgx              1/1     Running   0          52s
+kube-system   pod/kube-apiserver-u1a1                          1/1     Running   0          3m8s
+kube-system   pod/kube-apiserver-u1a2                          1/1     Running   0          2m37s
+kube-system   pod/kube-apiserver-u1a3                          1/1     Running   0          2m37s
+kube-system   pod/kube-controller-manager-u1a1                 1/1     Running   0          3m8s
+kube-system   pod/kube-controller-manager-u1a2                 1/1     Running   0          2m37s
+kube-system   pod/kube-controller-manager-u1a3                 1/1     Running   0          2m37s
+kube-system   pod/kube-proxy-2xxjh                             1/1     Running   0          108s
+kube-system   pod/kube-proxy-4rps2                             1/1     Running   0          2m44s
+kube-system   pod/kube-proxy-7zxbz                             1/1     Running   0          2m56s
+kube-system   pod/kube-proxy-h2xmz                             1/1     Running   0          108s
+kube-system   pod/kube-proxy-hm7lm                             1/1     Running   0          108s
+kube-system   pod/kube-proxy-jsxl6                             1/1     Running   0          2m44s
+kube-system   pod/kube-scheduler-u1a1                          1/1     Running   0          3m8s
+kube-system   pod/kube-scheduler-u1a2                          1/1     Running   0          2m37s
+kube-system   pod/kube-scheduler-u1a3                          1/1     Running   0          2m37s
+kube-system   pod/kubernetes-dashboard-84bfd98759-hsrjk        1/1     Running   0          50s
+kube-system   pod/kubernetes-metrics-scraper-79745547b-8g58c   1/1     Running   0          50s
+kube-system   pod/nginx-proxy-u1b1                             1/1     Running   0          107s
+kube-system   pod/nginx-proxy-u1b2                             1/1     Running   0          107s
+kube-system   pod/nginx-proxy-u1b3                             1/1     Running   0          107s
+kube-system   pod/nodelocaldns-4rdr7                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-fmb5b                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-kjx77                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-lr4cf                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-pvlwc                           1/1     Running   0          51s
+kube-system   pod/nodelocaldns-tjwfr                           1/1     Running   0          51s
+```
+
+### 4.2 OFFLINE / AIR-GAPPED
+
+- not fully supported / automated
```

Comment:
can you please provide links to resources explaining those two things?

Reply:
added definitions what "offline mode" and "airgapped mode" mean in this document and also some links to documentation